src/main/java/uk/gov/justice/digital/delius/controller/secure/AppointmentBookingController.java

```diff
@@ -48,4 +49,25 @@
         AppointmentCreateResponse response = appointmentService.createAppointment(crn, sentenceId, appointmentCreateRequest);
         return new ResponseEntity<>(response, HttpStatus.CREATED);
     }
+
+    @RequestMapping(value = "/offenders/crn/{crn}/sentence/{sentenceId}/well-known/appointments",
+                method = RequestMethod.POST,
+                consumes = "application/json")
```

Comment:
Not 100% sure of this but I wonder if this would have been better modelled as

`/offenders/crn/{crn}/sentence/{sentenceId}/appointments/context/:context`

So move context out of the body and as a parameter?
rather than "well-known".

Since isn't essentially  "I want to create a appointment using this context" ?

Reply:
We like that idea a lot. Will do